misc/scripts/build.sh

```diff
@@ -147,6 +147,10 @@ else
     -e "WORKER1=$WORKER1" \
     -e "WORKER2=$WORKER2" \
     -e "SSH_KEY_OPT=$SSH_KEY_OPT" \
+    -e "UPGRADE_FROM_VER=$UPGRADE_FROM_VER" \
+    -e "UPGRADE_TO_VER=$UPGRADE_TO_VER" \
+    -e "FROM_VIB_URL=$FROM_VIB_URL" \
```

Comment:
It's a bit painful (and unnecessary) to maintain the following two variables:
```sh
export FROM_VIB_URL=https://bintray.com/vmware/vDVS/download_file?file_path=VMWare_bootbank_esx-vmdkops-service_0.14.0577889-0.0.1.vib
export TO_VIB_URL=https://bintray.com/vmware/vDVS/download_file?file_path=VMWare_bootbank_esx-vmdkops-service_0.15.b93c186-0.0.1.vib
```
Since bintray download urls are not supposed to change, can we maintain a ver->url mapping in the code so that we just need two simple input variables (from_ver and to_ver) for the upgrade test?